docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,564 @@
+    "    print(\n",
+    "        f\"Mean rel. diff. between ideal and noisy results:\\n  {mean_vals}%\\n\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ccfbe164-34b0-4a21-9529-d4f015ca4a2b",
+   "metadata": {},
+   "source": [
+    "You can specify different noise models in the analyzer. For example, you can study the impact of depolarizing noise on our PUBs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "id": "d1051097-7111-498f-8177-3cbad849bd36",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Mean rel. diff. between ideal and noisy results:\n",
+      "  4.1%\n",
+      "\n",
+      "Mean rel. diff. between ideal and noisy results:\n",
+      "  7.88%\n",
+      "\n",
+      "Mean rel. diff. between ideal and noisy results:\n",
+      "  14.16%\n",
+      "\n",
+      "Mean rel. diff. between ideal and noisy results:\n",
+      "  17.22%\n",
+      "\n",
+      "Mean rel. diff. between ideal and noisy results:\n",
+      "  22.82%\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Set up a noise model with strength 0.02 on every two-qubit gate\n",
+    "noise_model = NoiseModel()\n",
+    "for qubits in backend.coupling_map:\n",
+    "    noise_model.add_quantum_error(\n",
+    "        depolarizing_error(0.02, 2), [\"ecr\", \"cx\"], qubits\n",
+    "    )\n",
+    "\n",
+    "# Update the analyzer's noise model\n",
+    "analyzer.noise_model = noise_model\n",
+    "\n",
+    "# Perform a noiseless simulation\n",
+    "ideal_results = analyzer.ideal_sim(clifford_pubs)\n",
+    "\n",
+    "# Perform a noisy simulation with the backend's noise model\n",
+    "noisy_results = analyzer.noisy_sim(clifford_pubs)\n",
+    "\n",
+    "# Compare the results\n",
+    "for idx, (ideal_res, noisy_res) in enumerate(\n",
+    "    zip(ideal_results, noisy_results)\n",
+    "):\n",
+    "    vals = rdiff(ideal_res, noisy_res)\n",
+    "\n",
+    "    # we print the mean relative difference for the various observables\n",
+    "    mean_vals = np.round(np.mean(vals), 2)\n",
+    "    print(\n",
+    "        f\"Mean rel. diff. between ideal and noisy results:\\n  {mean_vals}%\\n\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ce26d190-965c-4291-905c-959975f5f5dc",
+   "metadata": {},
+   "source": [
+    "As shown, given a noise model, one can try to quantify the impact of noise on the (Cliffordized version of the) PUBs of interest before running them on a QPU."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2037373d-686f-47ba-9516-ae8cd4a0659c",
+   "metadata": {},
+   "source": [
+    "### Application 2. Benchmark different strategies\n",
+    "\n",
+    "This example uses `Neat` to help identify the best options for our PUBs. To do so, consider running an estimation problem with PEA. Which noise amplification factors will  work best?"
```

Comment:
```suggestion
    "This example uses `Neat` to help identify the best options for your PUBs. To do so, consider running an estimation problem with PEA. Which noise amplification factors will  work best?"
```

Reply:
Maybe link out to the PEA page? https://docs.quantum.ibm.com/guides/error-mitigation-and-suppression-techniques#probabilistic-error-amplification-pea